COPYING

```diff
@@ -5,6 +5,6 @@ Redistribution and use in source and binary forms, with or without modification,
 
     * Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
```

Comment:
What are we doing about this line? 

```
Copyright (c) 2006-2020, Salvatore Sanfilippo
```

Reply:
That stays, we can't "revert" salvatore's copyright.